src/styles/normalize.scss

```diff
@@ -4,3 +4,7 @@ html {
   line-height: 13px;
   font-weight: 400;
 }
+
+a {
+  transition: color 0.4s, background-color 0.4s;
+}
```

Comment:
it's better to add transition to specific elements (links and buttons) using classes, not to all anchor tags